src/locales/en-AU.json

```diff
@@ -508,6 +509,7 @@
   "mnoe_admin_panel.dashboard.provisioning.subscriptions.cancel": "Cancel",
   "mnoe_admin_panel.dashboard.provisioning.subscriptions.reactivate": "Reactivate",
   "mnoe_admin_panel.dashboard.subscriptions.widget.list.title": "Orders",
+  "mnoe_admin_panel.dashboard.subscriptions.widget.list.cart_title": "Company Cart",
```

Comment:
Is this translation used?

Reply:
Yes it is sent from [organization.html](https://github.com/maestrano/mnoe-admin-panel/pull/267/files#diff-2a7bd634dee2b9d773b38450ee04e94cR201) to [mnoe-subscriptions-list component](https://github.com/MAhsenArif/mnoe-admin-panel/blob/7cc23308ffcef135a35d99e1dd0761a7fee6d2f9/src/app/components/mnoe-subscriptions-list/mnoe-subscriptions-list.coffee#L35)